spec/views/pages/homepage.html.erb_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe "pages/homepage.html.erb", type: :view do
+
```

Comment:
and, btw, with that beautiful integration test, you could skip this unit test (but you don't have to)